doc/BGP/BGP-route-aggregation-with-bbr-awareness.md

```diff
@@ -120,8 +120,18 @@ module sonic-bgp-aggregate-address {
                 }
 
                 leaf as-set {
-                     type boolean;
-                     description "Set if include the AS set when advertising the aggregated address";
+                    type boolean;
+                    description "Set if include the AS set when advertising the aggregated address";
+                }
+
+                leaf aggregate-address-prefix-list {
+                    type string;
```

Comment:
Is there a constrain on prefix-list name? If yes, we'd better add regex constrain to ensure user input not broken the whole system.
If plain text usually infers injection risk, we need make sure this is valid prefix name strictly.

Reply:
I understand, so I think I can use regex `[0-9a-zA-Z-_]+`
to define a not empty string which can only contain numbers, alphabet, underscore, and short dash.

Reply:
Updated